slices/util-linux.yaml

```diff
@@ -0,0 +1,271 @@
+package: util-linux
+
+essential:
+  - util-linux_copyright
+
+slices:
+  bins:
+    essential:
+      - util-linux_block-devices
+      - util-linux_cli-helpers
+      - util-linux_cpu
+      - util-linux_file-manipulation
+      - util-linux_file-system
+      - util-linux_generated
+      - util-linux_ipc
+      - util-linux_kernel
+      - util-linux_login
+      - util-linux_mcookie
+      - util-linux_memory
+      - util-linux_namespace
+      - util-linux_pam-profiles
+      - util-linux_process
+      - util-linux_set-arch
+      - util-linux_su-support
+      - util-linux_timer
+      - util-linux_tty
+
+  block-devices:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libmount1_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libudev1_libs
+    contents:
+      /usr/bin/lsblk:
+      /usr/bin/partx:
+      /usr/bin/resizepart:
+      /usr/sbin/blkdiscard:
+      /usr/sbin/blkid:
+      /usr/sbin/blkzone:
+      /usr/sbin/blockdev:
+      /usr/sbin/zramctl:
+
+  cli-helpers:
+    essential:
+      - libc6_libs
+      - libselinux1_libs
+    contents:
+      /usr/bin/getopt:
+      /usr/bin/namei:
+      /usr/bin/rev:
+      /usr/bin/whereis:
+
+  cpu:
+    essential:
+      - libc6_libs
+      - libsmartcols1_libs
+    contents:
+      /usr/bin/lscpu:
+      /usr/sbin/chcpu:
+
+  file-manipulation:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libmount1_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/fallocate:
+      /usr/bin/hardlink:
+      /usr/bin/more:
+      /usr/bin/rename.ul:
+
+  file-system:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libmount1_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libudev1_libs
+      - libuuid1_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/findmnt:
+      /usr/bin/mountpoint:
+      /usr/sbin/findfs:
+      /usr/sbin/fsck:
+      /usr/sbin/fsfreeze:
+      /usr/sbin/fstrim:
+      /usr/sbin/isosize:
+      /usr/sbin/mkfs:
+      /usr/sbin/mkswap:
+      /usr/sbin/pivot_root:
+      /usr/sbin/swaplabel:
+      /usr/sbin/switch_root:
+      /usr/sbin/wipefs:
+
+  # The following emulates the post-install script.
+  generated:
+    essential:
+      # Since the more binary is located at ./usr/bin/more and /bin in points
+      # to /usr/bin via base-files, the base-files_bin slice is included
+      # here.
```

Comment:
```suggestion
      # Since the more binary is located at ./usr/bin/more and /bin points
      # to /usr/bin via base-files, the base-files_bin slice is included
      # here.
```